cms/templates/js/certificate-details.underscore

```diff
@@ -0,0 +1,40 @@
+<div class="collection-details wrapper-certificate">
+    <header class="collection-header">
+        <h3 class="title">
+            <a href="#" class="toggle detail-toggle <% if (showDetails){ print('hide'); } else { print('show'); } %>-details">
```

Comment:
Lined 4 and 5 should be removed - no need to have collapsed state for this item.